src/redux/configureStore.js

```diff
@@ -0,0 +1,10 @@
+import { createStore, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
+import logger from 'redux-logger';
+import currencies from './currencies/currencies';
+
+const reducer = currencies;
+
+const store = createStore(reducer, applyMiddleware(thunk, logger));
```

Comment:
- [ ] Kindly remove the redux logger functionality or create a conditional so it doesn't show in production.

Reply:
Done.